src/components/autocomplete/autocomplete.spec.js

```diff
@@ -270,6 +270,85 @@ describe('<md-autocomplete>', function() {
       element.remove();
     }));
 
+    it('should use ng-repeat for standard-mode lists', inject(function() {
+      var scope = createScope(null, {menuContainerClass: 'custom-menu-container-class'});
+      var template = '\
+          <md-autocomplete\
+              md-menu-container-class="{{menuContainerClass}}"\
+              md-selected-item="selectedItem"\
+              md-search-text="searchText"\
+              md-items="item in match(searchText)"\
+              md-item-text="item.display"\
+              placeholder="placeholder"\
+              md-mode="standard">\
+            <span md-highlight-text="searchText">{{item.display}}</span>\
+          </md-autocomplete>';
+      var element = compile(template, scope);
+      var repeatingElement = element[0].querySelector('.md-standard-list-container li[ng-repeat]');
+      expect(repeatingElement).toBeDefined();
+
+      element.remove();
+    }));
+
+    it('should use md-virtual-repeat for virtual-mode lists', inject(function() {
+      var scope = createScope(null, {menuContainerClass: 'custom-menu-container-class'});
+      var template = '\
+          <md-autocomplete\
+              md-menu-container-class="{{menuContainerClass}}"\
+              md-selected-item="selectedItem"\
+              md-search-text="searchText"\
+              md-items="item in match(searchText)"\
+              md-item-text="item.display"\
+              placeholder="placeholder"\
+              md-mode="virtual">\
+            <span md-highlight-text="searchText">{{item.display}}</span>\
+          </md-autocomplete>';
+      var element = compile(template, scope);
+      var repeatingElement = element[0].querySelector('.md-virtual-repeat-container li[md-virtual-repeat]');
+      expect(repeatingElement).toBeDefined();
+
+      element.remove();
+    }));
+
+    it('should default to virtual-mode lists when md-mode is unspecified', inject(function() {
+      var scope = createScope(null, {menuContainerClass: 'custom-menu-container-class'});
+      var template = '\
+          <md-autocomplete\
+              md-menu-container-class="{{menuContainerClass}}"\
+              md-selected-item="selectedItem"\
+              md-search-text="searchText"\
+              md-items="item in match(searchText)"\
+              md-item-text="item.display"\
+              placeholder="placeholder">\
+            <span md-highlight-text="searchText">{{item.display}}</span>\
+          </md-autocomplete>';
+      var element = compile(template, scope);
+      var repeatingElement = element[0].querySelector('.md-virtual-repeat-container li[md-virtual-repeat]');
+      expect(repeatingElement).toBeDefined();
+
+      element.remove();
+    }));
+
+    it('should allow you to set a class to the standard mode .md-standard-list-container element', inject(function() {
```

Comment:
We don't currently strictly enforce this, but please try to keep line length under `100`.

Reply:
Done :+1: